src/WinRT.Runtime/ExceptionHelpers.cs

```diff
@@ -98,6 +98,34 @@ private static bool Initialize()
             return true;
         }
 
+        /// <summary>
+        /// Unhandled WinRT server exception event.
+        /// </summary>
+        public static event EventHandler<UnhandledWinRTServerExceptionEventArgs> UnhandledWinRTServerException;
+
+        public class UnhandledWinRTServerExceptionEventArgs : EventArgs
+        {
+            public Exception Exception { get; }
+            public bool Handled { get; set; }
+
+            public UnhandledWinRTServerExceptionEventArgs(Exception exception)
+            {
+                Exception = exception;
+            }
+        }
+
+        public static bool TryHandleWinRTServerException(object sender, Exception ex)
+        {
+            if (UnhandledWinRTServerException != null)
```

Comment:
Hoist this to a local to avoid the second static field fetch and also to guard against concurrent mutations.

Reply:
Done.